keras/backend/tensorflow_backend.py

```diff
@@ -3501,6 +3503,24 @@ def in_top_k(predictions, targets, k):
                           k=k)
 
 
+def top_k(input, k=1, sorted=True, name=None):
```

Comment:
Avoid adding new symbols to the backend (since they aren't present in tf.keras) and instead do an inline import of tf after checking `K.backend()`.

Reply:
Removed these symbols.